### DISCUSSION TITLE
How to change default edit-page icon?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4308] [Q&A] [answered]
Hi all,
the docs site has a custom edit-page icon as shown below

> <img width="878" alt="image" src="https://user-images.githubusercontent.com/5679861/187932109-f4eb2731-ba08-46f1-9a8d-fc0f97d39b8e.png">

How to choose that icon, instead of a default one which renders as this:

> <img width="866" alt="image" src="https://user-images.githubusercontent.com/5679861/187932201-ed3cd90d-85f3-4142-adce-3abd3057e137.png">

**Comment by squidfunk** [comment, marked as answer]
Already discussed in https://github.com/squidfunk/mkdocs-material/pull/4172#issuecomment-1197862055 – since nobody submitted a PR, but it keeps getting asked, I'll guess it's up to me to do it.

**Reply by squidfunk** [reply]
Added in 909591e0a. I've also added it to the JSON schema, but it still needs to be documented. I'll come back to that later.

In the meantime:

``` yaml
theme:
  icon:
    edit: material/file-edit-outline
```
